examples/mysql-wordpress-pd/README.md

```diff
@@ -278,6 +326,10 @@ NAME        CLUSTER-IP     EXTERNAL-IP     PORT(S)   AGE
 wordpress   10.0.0.5       1.2.3.4         80/TCP    19h
 ```
 
+**Note:** if you're using Host Path for storage then the wordpress service's
+`LoadBalancer` type is not implemented, and thus the wordpress pod's external
+IP address is not available (remains "Pending").
+
 ## Visit your new WordPress blog
```

Comment:
@jeffmendoza I will address your comments in the next push, thanks. I tried several ways to get an external IP visible but never saw an external IP for the wordpress service. Can you suggest a page to read or what needs to be done to get an external ip for wp? From my reading it appeared that `LoadBalancer` was tied to cloud providers.